lib/article_json/import/google_doc/html/text_box_parser.rb

```diff
@@ -22,10 +22,21 @@ def content
             @nodes.map { |node| parse_sub_node(node) }.compact
           end
 
+          # Extract any potential tags, specified in brackets after the Textbox definition
+          # @return [Array[Symbol]]
+          def tags
+            match = /(.*?)[\s\u00A0]+\[(?<tags>.*)\]/.match(@float_node.inner_text)
+            (match ? match[:tags] : '').split(' ')
```

Comment:
Not a blocker, but I'd prefer a guard clause to make the logic a bit easier to grasp:
```suggestion
            return [] unless match
            match[:tags].split(' ')
```